customers.sql

```diff
@@ -0,0 +1,8 @@
+select 
+o.customer_id, c.name, c.email, min(o.created_at) as first_order_at, count(o.id) as number_of_orders
+from `analytics-engineers-club.coffee_shop.customers` c
+join `analytics-engineers-club.coffee_shop.orders` o
```

Comment:
I don't like using single letters in aliases -- it's less legible/harder to grok.

Reply:
agree.. honestly, I wasn't planning to alias these tables at all but Big Query requires it even if you use the table name??? or maybe it was because I hadn't set the schema... I wonder....